api/openapi.yaml

```diff
@@ -0,0 +1,760 @@
+      parameters:
+        - $ref: "#/components/parameters/organizationId"
+        - $ref: "#/components/parameters/placeId"
+      responses:
+        "200":
+          description: Success, the number of users has been received.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PlaceUserCounter"
+        "404":
+          description: Place inside the organization not found.
+
+  # TODO This endpoint should probably also work with public organizations and non-anonymous users --> Alberto G. said: "According to the CAPITOLATO, you must monitor only the employee of a private organization. An other consideration: an anonymous user can't be recognizable by an userId, at least by the system (except InfluxDB)"
+  /monitor/{userId}:
+    get:
+      tags:
+        - stalker web app
+        - monitor
+        - organizations
+      description: Get all the data concerning the accesses of a specific employee. Operation permitted only if the user is inside a private organization, and reserved only to the viewers and up.
+      operationId: getAccessInformationByUserId
+      parameters:
+        - $ref: "#/components/parameters/userId"
+      responses:
+        "200":
+          description: Success, the informations about the user has been received.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/OrganizationUserCounter"
+                  - $ref: "#/components/schemas/PlaceUserCounter"
+        "404":
+          description: "Operation not allowed: the specified user is inside a public organization."
+
+  # FIXME There should be another endpoint concerning the entire report of user accesses, but this requirement is optional. So, go on
+
+  #FIXME? unique endpoint to describe adding/deleting web app users (i.e. an admin create/delete an owner exc.)
+  /user/{userId}/createUser:
+    post:
+      tags:
+        - stalker web app
+        - users
+      description: Create web app users (owners, managers, viewers). Operation reserved only to the administrator.
+      # TODO an alternative description
+      # description: Create all new users for an organization.
+      operationId: createNewUserInAnOrganization
+      parameters:
+        - $ref: "#/components/parameters/userId"
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/User"
+      responses:
+        "200":
+          description: Success, a new user has been created.
+        "404":
+          description: "Operation not allowed: the current user has no creation permission."
+        "408":
+          description: Request timed out.
+
+  /user/{userId}/deleteUser:
+    delete:
+      tags:
+        - stalker web app
+        - users
+      description: Remove all users for an organization.
```

Comment:
remove an user? O tutti?

Reply:
Remove an user, in quanto {userId} indica che va rimosso SOLO uno specifico utente.  Modificheremo la description